### PR TITLE
Drop similiar console tests

### PR DESCRIPTION
**Pull request by xpivarc**
**What this PR does / why we need it**:
Clean up tests that do the same thing.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by dhiller** [comment]
Hey @xpivarc , not sure why this fell off the plate somehow? Do you still want to pursue this?

**Comment by xpivarc** [comment]
> Hey @xpivarc , not sure why this fell off the plate somehow? Do you still want to pursue this?

@dhiller There is a failing test that I need to have a look at.

**Review by orelmisan — APPROVED** [review]
Thank you

File: tests/compute/console_test.go [hunk 1]
```diff
@@ -134,7 +117,13 @@ var _ = SIGDescribe("[rfe_id:127][posneg:negative][crit:medium][vendor:cnv-qe@re
...
 
 			It("[test_id:1593]should not be connected if scheduled to non-existing host", func() {
-				vmi := libvmi.NewAlpine(withNodeAffinityTo("kubernetes.io/hostname", "nonexistent"))
+				vmi := libvmi.NewAlpine(
+					libvmi.WithNodeAffinityFor(&k8sv1.Node{
+						ObjectMeta: metav1.ObjectMeta{
+							Name: "nonexistent",
+						},
+					}),
+				)
```
Comment (fossedihelm) [inline tests/compute/console_test.go:128]: This trick could be avoided with https://github.com/kubevirt/kubevirt/pull/10973/commits/4a773e5eb9aac52d98cf2ac875d69430ec521136

Comment (xpivarc) [reply tests/compute/console_test.go:128]: I agree, I can rebase if it gets in before this one.

**Pull request merged**